tools/anaconda-env/environment-MAC-arm64.yml

```
@@ -0,0 +1,32 @@
+name: nwb-guide
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python = 3.10.12
+  - PyInstaller = 5.13.0
+  - nodejs = 18.16.1
+  # - git = 2.41.0
+  - numcodecs = 0.11.0
+  - lxml = 4.9.3  # pypi build fails due to x64/arm64 mismatch so install from conda-forge
+  - pyedflib = 0.1.32  # pypi build fails due to x64/arm64 mismatch so install from conda-forge
+  - jsonschema = 4.18.0  # this installs jsonschema-specifications which is needed for validation
+  - pydantic[email] = 2.0.2  # email validator is used by dandi
+  # - opencv = 4.7.0
+  - numpy  # may have x64/arm64 mismatch issues so install from conda-forge
+  - pip
+  - pip:
+      - chardet == 5.1.0
+      # - openpyxl == 3.1.2
+      # - pennsieve == 6.2.0
+      - configparser == 6.0.0
+      # - python-docx == 0.8.11
+      # - xlrd == 2.0.1
+      # - biopython == 1.81
+      # - gevent == 22.10.2
+      - flask == 2.3.2
+      - flask_restx == 1.1.0
+      - flask-cors == 4.0.0
+      # - protobuf == 3.20.0
```

```suggestion
  - numcodecs = 0.11.0
  - lxml = 4.9.3  # pypi build fails due to x64/arm64 mismatch so install from conda-forge
  - pyedflib = 0.1.32  # pypi build fails due to x64/arm64 mismatch so install from conda-forge
  - jsonschema = 4.18.0  # this installs jsonschema-specifications which is needed for validation
  - pydantic[email] = 2.0.2  # email validator is used by dandi
  - numpy  # may have x64/arm64 mismatch issues so install from conda-forge
  - pip
  - pip:
      - chardet == 5.1.0
      - configparser == 6.0.0
      - flask == 2.3.2
      - flask_restx == 1.1.0
      - flask-cors == 4.0.0
```

cleanup - M1 build worked fine without these